sample/SkyApm.Sample.Logging/SkyApm.Sample.Logging.csproj

```diff
@@ -2,7 +2,7 @@
     <PropertyGroup>
         <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
         <TargetFramework>net7.0</TargetFramework>
-        <Nullable>enable</Nullable>
```

Comment:
What does this mean?

Reply:
[enable](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-reference-types)